packaging/debian/debian/rules

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	# nothing to build
+
+#override_dh_installsystemd:
```

Comment:
Debian experimental has its own [cryptsetup-suspend](https://salsa.debian.org/cryptsetup-team/cryptsetup/-/tree/debian/experimental/debian/scripts/suspend) implementation which should be compatible with ykfde. For this reason I don't think we ever implement it internally thus all references to suspend in debian files can be removed.

Reply:
That's good to know. I removed all reference to suspend from the debian files.